src/stores/user.js

```diff
@@ -113,13 +138,13 @@ export const useUserStore = defineStore('user', {
     },
 
     setProfileTab(tab) {
-      this.$patch({ _profileTab: tab })
+      this.$patch({_profileTab: tab})
     },
 
     async testing_loadUserProfile(user) {
       await getDoc(doc(db, 'users', user.uid)).then((document) => {
-        this.$patch({ _user: { uid: document.id, ...document.data() } })
```

Comment:
I'm noticing this throughout. The addition of extra space. Is this a config in one of ".editorconfig", ".eslintrc.js", or ".prettierrc"? I don't see it if it is but maybe I'm missing something.

Reply:
Can't use VSCode